libs/models/src/lib/sockets/tournament.ts

```diff
@@ -37,6 +52,12 @@ export enum TournamentState {
   Empty = 'empty',
 }
 
+// ==== Types ==== //
+/** Tournament id template literal */
+export type tournamentId = z.input<typeof tournamentIdSchema>;
```

Comment:
Please rename this to `TournamentId`. We need types to be in PsacalCase

Reply:
fixed de1f40717ab164f66408e3bf2acbe282ff4ce7e7

Reply:
Wrong commit ref